checklist_plus/perturb.py

```
@@ -584,3 +584,82 @@ def change_number(doc, meta=False, seed=None, n=10):
+        else:
+            text_str = str(text)
+
+        try:
+            # Use LLM to generate negated version
+            negated_versions = llm_text_generator.negate_sentence(
+                text_str,
+                n_variations=1,
+                **kwargs
+            )
+            return negated_versions[0] if negated_versions else None
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)